cirq-superstaq/cirq_superstaq/service.py

```diff
@@ -206,7 +206,8 @@ def get_counts(
         param_resolver: cirq.ParamResolverOrSimilarType = cirq.ParamResolver({}),
         method: str | None = None,
         **kwargs: Any,
-    ) -> dict[str, int]: ...
+    ) -> dict[str, int]:
+        ...
```

Comment:
it looks like your local version of `black` is out of date - once you update it and re-run the formatter these changes will be reverted